packages/connect-ui/package.json

```diff
@@ -10,7 +10,7 @@
         "serve:unsafe": "serve dist -p ${NANGO_CONNECT_UI_PORT:-3009}"
     },
     "dependencies": {
-        "serve": "14.2.4"
+        "serve": "^14.2.5"
```

Comment:
I thought we had agreed to pin our versions.

Reply:
Yeah, this was the direct audit suggestion. Still verifying this change on my local. I don't think I requested review yet but if I did I didn't mean to!